Brandon_hopper.asm

```diff
@@ -0,0 +1,228 @@
+# COMP3410 Program Template
+# Author: Brandon Hopper
+# Assignment: PA[X]
+# Date: 02/11/2015
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+
+Ask_Input: .asciiz "\How many numbers would you like to print?\n" 
+insert_into: .word 4 #Load word into memory
+buffer: .space 80
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+		
+	     la   $s0, fibs        # load address of array into $s0
+		la   $s5, size        # load address of size variable into $s5
+		
+	      #                 ------------My code below-------------
+	      #--------------------------------------------------------------------
+	      
+	      #Print message to ask for user input
+	      li $v0, 4
+	      la $a0, Ask_Input
+	      syscall
+		
+	      #Prompt for user input
+	      li $v0,5
+	      syscall
+              #move $s5, $v0
+	      
```

Comment:
-20 You are prompting for input and storing the input, but beyond that nothing else.  If you had uncommented line 45 and commented line 49 then the program would basically work but you would still need to check for valid input.  As it is now, it doesn't matter what the user enters because the program just goes back into its original code and prints 9 regardless.